agr_java_core/src/main/java/org/alliancegenome/core/service/OrthologyService.java

```diff
@@ -202,15 +224,19 @@ private static void putGeneInfo(Map<String, Object> map, Gene gene) {
         map.put(gene.getPrimaryKey(), data);
     }
 
-    public static JsonResultResponse<OrthologView> getOrthologyGenes(List<String> geneIDList, OrthologyFilter filter) {
+    public static JsonResultResponse<OrthologView> getOrthologyGenes(List<String> geneIDList, OrthologyFilter orthoFilter) {
         GeneCacheRepository repo = new GeneCacheRepository();
         List<OrthologView> orthologViewList = repo.getAllOrthologyGenes(geneIDList);
         List<OrthologView> filteredOrthologViewList = orthologViewList;
-        if (filter.getStringency() != null && !filter.getStringency().equals(OrthologyFilter.Stringency.ALL)) {
-            filteredOrthologViewList = orthologViewList.stream()
-                    .filter(orthologView -> orthologView.getStringencyFilter().equalsIgnoreCase(filter.getStringency().name()))
+
+        System.out.println("Filter: " + orthoFilter.getStringency().name());
```

Comment:
We use a logger for outputting info from the applications... please change the System.out.println -> log.info or log.debug thanks.

Reply:
In order to get a logger into the class just add the following annotation to the class:

`@Log4j2`
`public class OrthologyService {`

Reply:
I had used that merely for my own debugging purposes ..I took it out (but accidentally merged my pr :(

Reply:
Its all good...